src/tests/fixtures/externals/fundamentalErrors.ts.expected

```diff
@@ -0,0 +1,49 @@
+-----------------
+INPUT
+----------------- 
+// { "EXPERIMENTAL__emitResolverMap": false }
+
+import {
+  SomeType as _SomeType,
+  SomeInterface as _SomeInterface,
+} from "./nonGratsPackage.ignore";
+
+/**
+ * @gqlType MyType
+ * @gqlExternal "./test-sdl.ignore.graphql"
+ */
+export type SomeType = _SomeType;
+
+/**
+ * @gqlType MyType
+ * @gqlExternal
```

Comment:
Should this error since there's no schema path specified? I don't see an error for this in the output below.